lib/fireblocks/api/get_transaction_by_id.go

```diff
@@ -0,0 +1,36 @@
+package fireblocks
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/omni-network/omni/lib/errors"
+)
+
+func (c *Client) GetTransactionByID(ctx context.Context, transactionID string) (*TransactionResponse, error) {
+	var res TransactionResponse
+	endpoint := fmt.Sprintf("%s/%s", TransactionEndpoint, transactionID)
```

Comment:
the multiple nested wrapping of endpoint isn't great, lets make it super clear what path we querying by creating it in one place.